### DISCUSSION TITLE
How to trigger a Next.js build in Now using webhooks

### DISCUSSION THREAD

**Discussion by adamsoffer** [vercel/next.js#10808] [Help] [answered]
What's the recommended way to trigger a next.js build using webhooks in Now? For context, I'd like to deploy a CMS-powered next.js jamstack site to Now.

**Comment by Timer** [comment, marked as answer]
The ZEIT documentation explains this: https://zeit.co/docs/v2/more/deploy-hooks

Related: https://github.com/zeit/next.js/issues/9524

**Reply by adamsoffer** [reply]
Thanks @Timer. Just what I was looking for. And yeah, I'm using the new `getStaticProps` and `getStaticPaths` methods. So good.
